charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,100 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team acts as a liaison between labs who want to contribute data to the Human Cell Atlas (HCA) and the Data Coordination Platform (DCP). This team works with data contributing labs, to understand each lab's data and ensure the experiments are well-represented in the platform and to support the lab writing data to the platform, working with them to resolve any issues in the process.
```

Comment:
What does *experiments are well-represented* mean in this context? Good? Sufficient?

Editorial thoughts:

_The Ingest Support team is the liaison between the Data Coordination Platform and labs contributing data to the Human Cell Atlas (HCA). This team closely collaborates with each data contributor to:_

•	_Develop a deep understanding of their lab data_
•	_Ensure that their experiments are well-represented in the DCP_
•	_Help submit their lab data to the DCP, resolving any emerging issues during the process_